operator-framework-core/src/main/java/io/javaoperatorsdk/operator/processing/event/EventSourceManager.java

```diff
@@ -62,20 +59,22 @@ private void postProcessDefaultEventSources() {
    */
   @Override
   public synchronized void start() {
-    for (var eventSource : eventSources) {
-      try {
-        logEventSourceEvent(eventSource, "Starting");
-        eventSource.start();
-        logEventSourceEvent(eventSource, "Started");
-      } catch (MissingCRDException e) {
-        throw e; // leave untouched
-      } catch (Exception e) {
-        throw new OperatorException("Couldn't start source " + eventSource.name(), e);
-      }
-    }
+    startEventSource(eventSources.namedControllerResourceEventSource());
+    eventSources.additionalNamedEventSources().parallel().forEach(this::startEventSource);
     eventProcessor.start();
   }
 
+  @SuppressWarnings("rawtypes")
+
+
+  @Override
+  public synchronized void stop() {
+    stopEventSource(eventSources.namedControllerResourceEventSource());
```

Comment:
Shouldn't we stop the controller event source last?

Reply:
Was thinking about this, but at this does not matter.

Thinking rather that the processor should be stopped first. Before event sources.